fastlite/kw.py

```diff
@@ -172,7 +176,8 @@ def insert(
     columns: Union[Dict[str, Any], Default, None]=DEFAULT,
     strict: opt_bool=DEFAULT,
     **kwargs) -> Table:
-    if not record: record={}
+    if not kwargs and not record: return {}
```

Comment:
If you moved this a couple of lines down I think you can just do “if record”?

Reply:
That doesn't work, it breaks the tests and behavior. Perhaps we're missing a detail?

Reply:
After looking at it again, we addressed this, reducing line count in the method.